Makefile

```diff
@@ -19,6 +20,9 @@ push/3scale:
 	operator-courier verify 3scale
 	-operator-courier push 3scale/ $(REPO) 3scale $(3SCALE_VERSION) "$(AUTH_TOKEN)"
 
+push/launcher:
+	operator-courier verify launcher
+	-operator-courier push launcher/ $(REPO) launcher $(LAUNCHER_VERSION) "$(AUTH_TOKEN)"
```

Comment:
this line has changed format recently, can you copy it from the new format on master